signbank/dictionary/forms.py

```diff
@@ -102,6 +102,15 @@ class GlossSearchForm(forms.ModelForm):
     hasnovideo = forms.BooleanField(label=_('No videos'), required=False)
     multiplevideos = forms.BooleanField(label=_('Multiple videos'), required=False)
 
+    # Creating choices for related to foreign sign list
+    related_to = [(0, '---------')]
```

Comment:
I think if you specify`None` here, the option will just have no value, which is what you want, and you can then simplify the `if` condition in `adminviews.py`. It's not really a good idea to have lots of different things which mean the same as `None`